test/Npgsql.Tests/YBFallbackTopolgyTests.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Npgsql.Tests;
+
+public class YBFallbackTopolgyTests
+{
+    static int mlock = 0;
+    [Test]
+    public async Task TestFallback()
+    {
+        var connStringBuilder = "host=127.0.0.3;port=5433;database=yugabyte;userid=yugabyte;password=yugsbyte;Load Balance Hosts=true;Timeout=0;Topology Keys=";
+        var conns = await CreateConnections(connStringBuilder+"aws.us-west.us-west-2a,aws.us-west.us-west-2c", 6, 0, 6);
+        CloseConnections(conns);
+        conns = await CreateConnections(connStringBuilder + "aws.us-west.us-west-2a,aws.us-west.us-west-2b:1,aws.us-west.us-west-2c:2", 6, 6, 0);
+        CloseConnections(conns);
+        // await CreateConnections(connStringBuilder + "aws.us-west.us-west-2a:1,aws.us-west.us-west-2b:2,aws.us-west.us-west-2c:3", 12, 0, 0);
```

Comment:
Why are these cases commented out?

Reply:
Uncommented them , and made them into separate cases